docs/create-new-environment.md

```diff
@@ -0,0 +1,127 @@
+# Sparkle Web App
+
+Codebase for the Sparkle platform, brought to you by [Sparkle](https://sparklespaces.com/), a platform for the most immersive & interactive virtual events.
+
```

Comment:
Don't need this header info stuff, it's in the main README

```suggestion
```

Reply:
025e814